packages/esm-patient-biometrics-app/src/biometrics/weight-tile.component.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useBiometrics } from './biometrics.resource';
+import { useTranslation } from 'react-i18next';
+import { useVitalsConceptMetadata } from '@openmrs/esm-patient-common-lib';
+import styles from './weight-tile.component.scss';
+
+interface WeightTileInterface {
+  patientUuid: string;
+}
+
+const WeightTile: React.FC<WeightTileInterface> = ({ patientUuid }) => {
+  const { data: biometrics } = useBiometrics(patientUuid);
+  const { data: conceptUnits } = useVitalsConceptMetadata();
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.gridColumn}>
+      <p>{t('weight', 'Weight')}</p>
+      <span>
+        {biometrics?.[0].weight} {biometrics?.[0].weight && conceptUnits?.conceptUnits[4]}
```

Comment:
This will just be the weight from the last biometrics/vitals, right? I think it would be better if it was the most recent weight, in case weight wasn't taken the last time the form was filled.